src/react/fileUrlToFileUrlObject.ts

```diff
@@ -47,7 +78,7 @@ const getFileUrlObject = ({
       remainingPathParts.length === 0 ||
       child.children?.length
     ) {
-      if (child.name === nextPathPart) {
+      if (child.name === nextPathPart || child?.paths?.[locale] === nextPathPart) {
```

Comment:
Idem.

Reply:
We need to localise the paths as explained in my previous comments 😄